cli/running/writer_test.go

```diff
@@ -0,0 +1,64 @@
+package running
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func getLogsBytes(logs []string) []byte {
+	return []byte(strings.Join(logs, "\n"))
+}
+
+func getLogsWithPrefix(prefix string, logs []string) string {
+	resLines := make([]string, len(logs))
+	for i := range logs {
+		resLines[i] = fmt.Sprintf("%s | %s", prefix, logs[i])
+	}
+
+	return strings.Join(resLines, "\n")
+}
+
+func TestWrite(t *testing.T) {
+	t.Parallel()
+
+	assert := assert.New(t)
+
+	out := bytes.NewBuffer(nil)
+	var logs []string
+	var logBytes []byte
+	var n int
+	var err error
```

Comment:
This is looks like a Pascal-style: variables are declared at the beginning, not in the place where they will be used

Reply:
See below, this variables are used on each test.